src/login/components/LoginApp/ResetPassword/GoBackButton.tsx

```diff
@@ -29,12 +29,10 @@ export function GoBackButton(props: BackToLoginButtonProps): JSX.Element | null
     }
   }
 
-  const style = props.primary ? "primary" : "secondary";
+  const style = props.primary ? "primary" : props.secondary ? "secondary" : "link";
```

Comment:
Consider replacing props.primary and props.secondary with just props.style passed on to EduIDButton.